modules/spo-applying-profiles.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-applying-profiles_{context}"]
+= Applying {type} profiles to a pod
+
+Create a Pod to apply one of the created profiles.
+
+.Procedure
+
+ifdef::seccomp[]
+. Create a Pod object that defines a `securityContext`:
```

Comment:
Do you have a trace?

Reply:
I would check this:
1. does a seccomp profile exist that has the usage of `operator/my-namespace/profile1.json` ?
2. does the my-namespace ns exist?
3. are you creating the pod in my-namespace (the manifest does not say)
4. by default, OCP doesn't allow pods running as root  and the nginx image relies on running as root. This can be solved using a non-default SCC and probably we should add an explicit example that shows how to run a root container (even though it's a bit besides the point of SPO). I would try a different image.

Here's a complete example that works for me:
```
oc create ns nginx-seccomp
oc project nginx-seccomp
oc create -f examples/seccompprofile.yaml
oc get seccompprofile.security-profiles-operator.x-k8s.io/profile-allow-unsafe -oyaml
[...]
status:
  conditions:
  - lastTransitionTime: "2022-11-15T13:43:55Z"
    reason: Available
    status: "True"
    type: Ready
  localhostProfile: operator/nginx-seccomp/profile-allow-unsafe.json
cat pod.yaml
apiVersion: v1
kind: Pod
metadata:
  name: test-pod
  namespace: nginx-seccomp
spec:
  securityContext:
    seccompProfile:
      type: Localhost
      localhostProfile: operator/nginx-seccomp/profile-allow-unsafe.json
  containers:
    - name: nginx
      image: quay.io/security-profiles-operator/test-nginx-unprivileged:1.21
oc create -f pod.yaml
oc get pods test-pod -ojsonpath='{.spec.securityContext.seccompProfile}'
{"localhostProfile":"operator/nginx-seccomp/profile-allow-unsafe.json","type":"Localhost"}
```

Reply:
I'll give it a shot, thanks!